src/ApparentHorizons/Tags.hpp

```diff
@@ -12,7 +12,19 @@
 #include "Utilities/ForceInline.hpp"
 #include "Utilities/TMPL.hpp"
 
+/// \cond
 class DataVector;
+class FastFlow;
+/// \endcond
+
+namespace ah {
+namespace Tags {
+struct FastFlow : db::SimpleTag {
+  static std::string name() noexcept { return "FastFlow"; }
```

Comment:
For this tag you should just test the name, as it is done for other tags in the code

Reply:
Done.